lib/helpers/markdown/plugins/video.js

```diff
@@ -90,6 +90,11 @@ function render(tokens) {
       embedURL = `//player.vimeo.com/video/${uri.path.split('/').pop()}`;
     }
 
+    // Ustream URL
+    else if (uri.hostname === 'www.ustream.tv') {
+      embedURL = `http://www.ustream.tv/embed/recorded/${uri.path.split('/').pop()}`;
```

Comment:
* The embed URL should either hardcode the `https` URL or it should use the protocol independent url (`//`). It shouldn't hardcode the unsecure HTTP URL
* While this renders as expected, this doesn't appear to be the correct embed URL for this video. The embed URL appears to be `//www.ustream.tv/embed/${id}` for… I think… live? videos and the `//www.ustream.tv/embed/recorded/${id}` URL for previously recorded videos. This needs to be able to support both `live` and `recorded` videos